### PR TITLE
Add documentation and validate reStructuredText markup

### PR DESCRIPTION
**Pull request by aitordiaz**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,76 +1,231 @@
...
 
+.. image:: https://readthedocs.org/projects/averell/badge/?style=flat
+    :target: https://readthedocs.org/projects/averell
+    :alt: Documentation Status
 
 .. end-badges
 
-Corpora downloader and reader for Spanish sources
+Averell, the python library and command line interface that facilitates working
+with existing repositories of an-notated poetry. \
```
Comment (alvp) [inline README.rst:22]: an-notated -> annotated

File: src/averell/utils.py [hunk 2]
```diff
@@ -288,9 +283,9 @@ def read_features(corpus_folder):
 
 def pretty_string(text, n_words):
     """Add a line break every number of words into a text to create multiline
-    cells to use in `tabulate_corpora`
+    cells to use in :py:func:`~averell.utils.get_main_corpora_info`
 
-    :param text: String to be converted
+    :param text: String to be divided
```
Comment (alvp) [inline src/averell/utils.py:288]: I think `to be split` is better

File: src/averell/utils.py [hunk 3]
```diff
@@ -288,9 +283,9 @@ def read_features(corpus_folder):
...
     :param n_words: Number of words to add a line break after
```
Comment (alvp) [inline src/averell/utils.py:289]: `n_word` is a term to refer to the word "nigger", not sure if this is appropiate :), maybe change it to something else?

File: README.rst (hunk 1 above)
Comment (aitordiaz) [reply README.rst:22]: Done

File: src/averell/utils.py (hunk 2 above)
Comment (aitordiaz) [reply src/averell/utils.py:288]: ✔️

File: src/averell/utils.py (hunk 3 above)
Comment (aitordiaz) [reply src/averell/utils.py:289]: Sorry, changed to num_words

**Pull request merged**